test/functional/rpc_getblockstats.py

```diff
@@ -49,7 +49,7 @@ def generate_test_data(self, filename):
         self.sync_all()
 
         self.nodes[0].sendtoaddress(address=address, amount=10, subtractfeefromamount=True)
-        self.nodes[0].sendtoaddress(address=address, amount=10, subtractfeefromamount=False)
+        self.nodes[0].sendtoaddress(address=address, amount=0.00001, subtractfeefromamount=False)
```

Comment:
Nice way to trigger the dust statistics!

Reply:
Can you add a comment to say that this amount is to trigger the dust statistics.